### PR TITLE
SIG Governance Requirements and Recommendations

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
This looks good to me.

File: committee-steering/governance/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Specific attention has been given to:
+
+- The role of technical leadership
+- The role of operational leadership
+- Process for agreeing upon technical decisions
+- Process for ensuring technical assets remain healthy
+
+## How to use the templates
+
+When developing or modifying a SIG governance doc, the intention is for SIGs to use the templates (*forthcoming*) as a [buffet]
```
Comment (quinton-hoole) [inline committee-steering/governance/README.md]: nit: I understand what you mean here by "buffet" but I think the term might be a bit opaque for some of our community.  Perhaps spell it out in more direct language?

File: committee-steering/governance/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+to choose options from.  It is recommended that SIGs start by looking at the
+[Recommendations and requirements] for SIG governance docs before selecting and defining strategies.
```
Comment (quinton-hoole) [inline committee-steering/governance/README.md]: Of course you need the link here.

File: committee-steering/governance/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+The expectation is that SIGs will adapt the provided templates as best to meet the needs of the both the SIG
+and project.
+
+- [Recommendations and requirements]
+
+[Recommendations and requirements]: sig-governance-requirements.md
```
Comment (quinton-hoole) [inline committee-steering/governance/README.md]: I guess the URL's need to be in () here.

File: committee-steering/governance/sig-governance-requirements.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+- *MAY* define process for new community members to contribute to the area
+  - e.g. read a contributing guide, show up at SIG meeting, message the google group
+
+### Project management
+
+- *MUST* define how subprojects are managed if at all
+  - When and how new subprojects are created
+  - Roles within subprojects
+- *MUST* define how priorities / commitments are managed
```
Comment (quinton-hoole) [inline committee-steering/governance/sig-governance-requirements.md]: I wonder whether this one would be better as "SHOULD"?

File: committee-steering/governance/sig-governance-requirements.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+  - What priorities are targeted for milestones
+  - The process for publishing a release
+
+### Technical processes
+
+For areas owning technical assets only.
+
+- *MUST* define how technical decisions are communicated and made within the SIG or project
+  - Process for proposal, where and how it is published and discussed, when and how a decision is made
+    (e.g. KEP process)
```
Comment (quinton-hoole) [inline committee-steering/governance/sig-governance-requirements.md]: Suggest adding a link for KEP.

File: committee-steering/governance/README.md (hunk 2 above)
Comment (pwittrock) [reply committee-steering/governance/README.md]: This should work by referencing the alias at the bottom.  It seems to work for my when I click 'View'.

File: committee-steering/governance/README.md (hunk 3 above)
Comment (pwittrock) [reply committee-steering/governance/README.md]: This is a shortcut:

https://sourceforge.net/p/aliascommand/home/markdown_syntax/#md_ex_reflinks

**Comment by pwittrock** [comment]
@quinton-hoole Thanks for the feedback.  Addressed.

File: committee-steering/governance/sig-governance-requirements.md (hunk 5 above)
Comment (pwittrock) [reply committee-steering/governance/sig-governance-requirements.md]: ACK

File: committee-steering/governance/sig-governance-requirements.md (hunk 4 above)
Comment (pwittrock) [reply committee-steering/governance/sig-governance-requirements.md]: ACK

File: committee-steering/governance/README.md (hunk 1 above)
Comment (pwittrock) [reply committee-steering/governance/README.md]: ACK

**Pull request merged**